docs/content_tabs.rst

```diff
@@ -0,0 +1,122 @@
+.. _sphinx-design tabs: https://sphinx-design.readthedocs.io/en/furo-theme/tabs.html
+
+Content tabs
+============
+
+.. note::
+    This document focused on content tabs, not navigation tabs.
+
+Use of `content tabs in mkdocs-material <https://squidfunk.github.io/mkdocs-material/reference/content-tabs/>`_
+theme relies on a markdown extension that isn't used in the world of Sphinx. Instead,
+the sphinx-immaterial theme provides its own directives to make use of content tabs.
+
+.. admonition:: Linked Tabs
+    :class: missing
+
+    The `linked content tabs <https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#linked-content-tabs>`_
+    seen in mkdocs-material is not supported until that feature transitions from the mkdocs-material theme's insider
+    releases to it's public releases.
+
+    However, you can use other sphinx extensions (like `sphinx-design tabs`_) to achieve this functionality.
+    Although, other extensions will require some custom CSS styling to match the mkdocs-material
+    theme's styling for content tabs.
+
+.. confval:: md-tab-set
```

Comment:
Rather than confval, we should use the rst domain provided by sphinx:

https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#the-restructuredtext-domain

Reply:
Done.

This is the first time I've actually used the rst domain, so let me know if there are any tweaks you'd like to see,